ext/reflection/php_reflection.c

```diff
@@ -3654,6 +3654,22 @@ ZEND_METHOD(ReflectionMethod, getDeclaringClass)
 }
 /* }}} */
 
+/* {{{ Returns whether a method has a prototype or not */
+ZEND_METHOD(ReflectionMethod, hasPrototype)
+{
+    reflection_object *intern;
+    zend_function *mptr;
+
+    GET_REFLECTION_OBJECT_PTR(mptr);
```

Comment:
Usually, `GET_REFLECTION_OBJECT_PTR(mptr);` is called after param validation, but I see that the `getPrototype()` is different in this regard. I think it's a bad practice, so could you please change the order?

Reply:
@kocsismate do you want me to change it for `getPrototype` too?

Reply:
Let's just stick with the new one. And I'll commit it for `getPrototype` soon.